pkg/nsx/services/inventory/builder.go

```diff
@@ -435,9 +435,6 @@ func (s *InventoryService) determineServiceStatus(podIDs []string, hasAddr, hasS
 		}
 	} else if hasAddr {
 		status = InventoryStatusUnknown
-		if hasAnnotationError {
-			status = InventoryStatusDown
-		}
 		netStatus = NetworkStatusUnhealthy
```

Comment:
set netStatus to NetworkStatusUnhealthy only when hasAnnotationError

Reply:
ok

Reply:
Also line 447, if status is down, network status will always be unhealthy?